src/PhpDocument.php

```diff
@@ -176,7 +176,7 @@ public function parse()
         $diagnostics = [];
         foreach ($errors as $error) {
             $diagnostic = new Diagnostic();
-            $startLine = $error->getStartLine() === -1 ? 0 : $error->getStartLine() - 1;
+            $startLine = max($error->getStartLine() - 1, 0);
             $startColumn = $error->hasColumnInfo() ? $error->getStartColumn($this->content) - 1 : 0;
             $endLine = $error->getEndLine() === -1 ? $startLine : $error->getEndLine() - 1;
```

Comment:
Well one possible point of error you leave open is if it returns 0 (or `null`/`false`), because then we subtract 1 from it, and have `-1` again. I think the `max` approach is the safest.